content/en/database_monitoring/setup_oracle/rac.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Setting Up Database Monitoring for Oracle RAC
+kind: documentation
+description: Install and configure Database Monitoring for Oracle RAC
+private: true
+is_beta: true
+further_reading:
+- link: "/integrations/oracle/"
+  tag: "Documentation"
+  text: "Basic Oracle Integration"
+
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Database Monitoring is not supported for this site.</div>
+{{< /site-region >}}
+
+<div class="alert alert-info">
+The features described on this page are in private beta. Contact your Customer Success Manager for information on the required Agent build and installation instructions.
+</div>
+
+Database Monitoring provides deep visibility into your Oracle databases by exposing query samples to profile your different workloads and diagnose issues.
+
+## Before you begin
+
+Supported Oracle versions
+: 19c, 21c
+
+Supported Agent versions
+: For more information on the required Agent build, contact your Customer Success Manager.
```

Comment:
Same comment here, let's include the Agent build / installation instructions so that we don't have to be going back and forth with the CSMs

We can include a note saying something like, "as this feature is in private beta, new builds may be released, but the one listed here supports XYZ functionality"

Reply:
I specified only the latest Agent version now, because it doesn't make sense for a new customer to install an older version.